_examples/autobahn/server.go

```diff
@@ -84,7 +84,7 @@ func echoCopyFull(w http.ResponseWriter, r *http.Request) {
 }
 
 // echoReadAll echoes messages from the client by reading the entire message
-// with ioutil.ReadAll.
+// withio.ReadAll.
```

Comment:
fix typo, please!

Reply:
Fixed! I wonder how local go tests didn't catch that.

Reply:
Actually, there are a few other things I need to fix, so please wait as I fix them

Reply:
ok, now it should work